<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Tab_GerenciarProduto">
     <attribute name="title">
      <string>Gerenciar Produto</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonConsultar">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>391</width>
        <height>191</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditConsultar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>22</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Digite o código:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_22">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Código</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCodigoGe"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Descrição</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDescricaoGe"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Quantidade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditQuantidadeGe"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Preco</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>R$</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPrecoGe"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButtonCadastrarNovo">
           <property name="text">
            <string>Cadastrar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonExcluir">
           <property name="text">
            <string>Excluir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonListar">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSegredo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>200</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
     <zorder>lineEditSegredo</zorder>
     <zorder>pushButtonConsultar</zorder>
     <zorder>tableWidget</zorder>
     <zorder>lineEditConsultar</zorder>
     <zorder>label_21</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>pushButtonListar</zorder>
    </widget>
    <widget class="QWidget" name="NomeDaTab">
     <attribute name="title">
      <string>Gerenciar Cliente</string>
     </attribute>
     <widget class="QTextEdit" name="textEditRetirarCliente">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonConsultar_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>30</y>
        <width>269</width>
        <height>298</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Codigo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Cpf ou Cnpj:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLineEdit" name="lineEditCodigoCliente"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCpf_Cnpj_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Nome ou Razão Social:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditNome_Razao_Social_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Celular ou Nome Contato:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCelular_Nome_Contato_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Telefone:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTelefone_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEmail_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Logradouro:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditLogradouro_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Estado:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEstado_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Cidade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditCidade_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Setor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSetor_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>357</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar Cliente:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRegistrarCliente">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>0</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo Cliente:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Retirar Cliente:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRetirarCliente">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Retirar</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>30</y>
        <width>178</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_23">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Codigo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditConsultar_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>178</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Codigo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRetirarCliente_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonListarClientes">
      <property name="geometry">
       <rect>
        <x>394</x>
        <y>370</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar Clientes</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetPessoa">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>281</width>
        <height>221</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Codigo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome/Razão Social</string>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Celular/Nome Contato</string>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cpf/Cnpj</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Logradouro</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cidade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Setor</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_nome2">
     <attribute name="title">
      <string>Gereciar Pedido</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Pedido: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Cliente:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPedido">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCliente">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>CADASTRAR:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CADASTRAR ITENS DO PEDIDO:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Produto:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProduto">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantidade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditQuantidade">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCadastrarItem">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadastrar Item</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor total do item:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPreco">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor total do pedido:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTotalPedido">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CONSULTAR:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Pedido: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPedidoCodigoConsultar">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditConsultar_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>381</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVER PEDIDO: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonConsultar_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRemover">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Pedido: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPedidoCodigoRemover">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditRemover">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LISTAR PEDIDOS:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditListar">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonListar_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Listar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
